README.md

```diff
@@ -45,21 +45,21 @@ items/[itemname].png
 -----------
 For each item in your resources.yaml file you will need to have an image for that resource. If an image is missing, then the build process will submit a warning and fill that space with a purple color, indicating it does not have an icon.
 The icons can be any size, even rectangular, but they all must be the same size as each other for a given calculator.
-The file names of each file should be the resource name in all lower case with no spaces or punctuation.  
+The file names of each file should be the resource name in all lower case with no spaces or punctuation.
 
-For Example:  
-*"Pink Stained Glass Pane"* becomes *[pinkstainedglasspane.png](resource_lists/minecraft/items/pinkstainedglasspane.png)*  
+For Example:
+*"Pink Stained Glass Pane"* becomes *[pinkstainedglasspane.png](resource_lists/minecraft/items/pinkstainedglasspane.png)*
 *"Jack 'o Lantern"* becomes *[jackolantern.png](resource_lists/minecraft/items/jackolantern.png)*
```

Comment:
I'm guessing you have a bad or misconfigured markdown linter. Two trailing spaces in markdown is a newline. Remove the two spaces on line 8 is fine because it goes right into a list which is already on a newline. But this change pushes this all onto one line.
Before:
![Screenshot from 2024-07-27 09-26-16](https://github.com/user-attachments/assets/9035c8ff-240e-431a-89fc-f4073b94c9a5)
After:
![Screenshot from 2024-07-27 09-25-53](https://github.com/user-attachments/assets/95cd79ff-cfe0-4144-9189-471c003de85c)

Reply:
Looks like they changed the setting from `"diffEditor.ignoreTrimWhitespace": true` to `"files.trimTrailingWhitespace": false` so it wasn't respecting that I had set md files to not have it removed.

Big fan of positively naming variables but annoying that they broke settings.